/*
  Vehicle Data

  This API section holds messages related to OTA Vehicle Data between device and backend.

  The version of the OpenAPI document: 3

  Generated by OpenAPI Generator: https://openapi-generator.tech
*/

syntax = "proto3";

package com.bosch.bfb.protobuf.vehicledata.v3;

import public "backend_to_device_message_meta.proto";

message ActivateMeasurementCommand {

  BackendToDeviceMessageMeta meta = 3347973;

  // Serialization method of measurementConfiguration, e.g. base64
  string serialization = 385936369;

  // Serialized measurement configuration
  string measurementConfiguration = 41663131;

  // Information about the stack used e.g. AA-Stack
  string stackType = 117326309;

}
